h3i/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "h3i"
+version = "0.1.0"
+edition = "2018"
+
```

Comment:
Need to add more metadata here to be able to publish to crates.io, e.g. see https://github.com/cloudflare/quiche/blob/master/quiche/Cargo.toml and https://github.com/cloudflare/quiche/blob/master/quiche/Cargo.toml